scripts/tests

```diff
@@ -416,8 +413,11 @@ def bench(
         else:
             exec_wrapper = f"taskpolicy -c {mac_taskpolicy}"
 
-    if exec_wrapper and " " in exec_wrapper:
-        exec_wrapper = exec_wrapper.split(" ")
+    if exec_wrapper:
+        if " " in exec_wrapper:
+            exec_wrapper = exec_wrapper.split(" ")
+        else:
+            exec_wrapper = [exec_wrapper]
```

Comment:
Nit: I think you can unconditionally do `exec_wrapper.split(" ")` as that will return a singleton list in case `" "` does not occur in `exec_wrapper.`

Reply:
true. Thanks